logstash-core/lib/logstash/instrument/periodic_poller/dlq.rb

```diff
@@ -10,10 +10,15 @@ def initialize(metric, agent, options = {})
     end
 
     def collect
-      _, pipeline = @agent.with_running_pipelines { |pipelines| pipelines.first }
-      unless pipeline.nil?
-        pipeline.collect_dlq_stats
+      pipelines = @agent.with_running_user_defined_pipelines {|pipelines| pipelines}
+      unless pipelines.nil?
```

Comment:
btw, you could write the pipelines.each block as a single line (if you need the nil? check at all)

```
pipelines.each { |_, pipeline| pipeline.collect_dlq_stats unless pipeline.nil? }
```

And I realize I didn't do that in the initial commit ;)